Languages/en.json

```diff
@@ -0,0 +1,131 @@
+"Console_Convertor_Settings_Error_GM_FileNotFound" : "{file_path} file not found.",
+"Console_Convertor_Settings_Error_yypNameNotRead" : "Error reading project name from .yyp file: {error}",
+"Console_Convertor_Settings_Error_yypGeneric" : "Error reading {option_name} from {os.path.basename(file_path)}: {error}",
+"Console_Convertor_Settings_Error_GD_NotFound" : "project.godot file not found in the Godot project directory.",
+"Console_Convertor_Settings_UpdatedName" : "Updated project name to: {gm_project_name}",
+"Console_Convertor_Settings_Error_Name_GM" : "Could not update project name: GameMaker project name not found.",
+"Console_Convertor_Settings_Error_NameGeneric" : "Error updating project name: {error)}.",
+"Console_Convertor_Settings_Updated" : "Updated project.godot with GameMaker settings",
+"Console_Convertor_Settings_Error_NotUpdated" : "Error updating project settings: {error}",
+
+"Console_Convertor_AudioBus" : "Generating audio bus layout...",
+"Console_Convertor_AudioBus_Error_SectionNotFound_GM" : "AudioGroups section not found in the .yyp file.",
+"Console_Convertor_AudioBus_Error_NameNotFound_GM" : "No AudioGroup names found in the .yyp file.",
+"Console_Convertor_AudioBus_Error_Group_Generic" : "Error reading AudioGroups from .yyp file: {error}",
+"Console_Convertor_AudioBus_Group_Found" : "Found AudioGroups: {audio_group_names}",
+"Console_Convertor_AudioBus_Group_Generated" : "Generated default_bus_layout.tres with {audio_groups_num} audio buses.",
+"Console_Convertor_AudioBus_Error_GroupNotGenerated" : "Error generating default_bus_layout.tres: {error}",
+
+"Console_Convertor_Sprites" : "Converting sprites...",
+"Console_Convertor_Sprites_Complete" : "Sprite conversion completed.",
+"Console_Convertor_Sprites_Stopped" : "Sprite conversion stopped.",
+"Console_Convertor_Sprites_Converted" : "Converted: {relative_path} -> sprites/{sprite_name}/{new_filename}",
+"Console_Convertor_Sprites_Error_NotFound" : "No sprite images found in the GameMaker project.",
+
+"Console_Convertor_Fonts" : "Converting fonts...",
+"Console_Convertor_Fonts_Complete" : "Font conversion completed.",
+"Console_Convertor_Fonts_Error_NotFound" : "Font folder not found in {gm_project_path}",
+
+"Console_Convertor_Tilesets" : "Converting tilesets...",
+"Console_Convertor_Tilesets_Complete" : "Tileset conversion completed.",
+"Console_Convertor_Tilesets_Error_NotFound" : "Tileset folder not found in {self.gm_project_path}",
+
+"Console_Convertor_Sounds" : "Converting sounds...",
+"Console_Convertor_Sounds_Converted" : "Converted: {path} -> sounds/{path}",
+"Console_Convertor_Sounds_Error_NotFound" : "No sound files found in the GameMaker project.",
+"Console_Convertor_Sounds_Complete" : "Sound conversion completed.",
+"Console_Convertor_Sounds_Stopped" : "Sound conversion stopped.",
+
+"Console_Convertor_Shaders_Converted" : "Converted {filename} to {output_path}",
```

Comment:
I think we can use similar syntax for Converting **something to something**. Here we use `to`, but in `Console_Convertor_Sounds_Converted` we use `->`.
@Infiland what do you think?